app/components/post_draft/post_input/post_input.tsx

```diff
@@ -295,6 +297,18 @@ export default function PostInput({
         };
     }, [handleHardwareEnterPress]);
 
+    /**
+     * Auto-focus input if hardware keyboard is connected.
+     */
+    useEffect(() => {
+        (async () => {
+            const isHwKeyboardActive = await MattermostManaged.isHwKeyboardActive();
```

Comment:
This will crash android as the function is not defined

Reply:
Ah, good catch. Can you please advice on what the best practice here would be? Should I add a no-op equivalent for Android, or do I use a `Platform.OS` check instead?

Reply:
I rather have a fully functional op on Android as well